app/app.yaml

```diff
@@ -0,0 +1,39 @@
+components:
+  schemas:
+    App:
+      description: >-
+        A container for emulated protocol and traffic configuration.
+      type: object
+      properties:
+        tcp:
+          description: >-
+            The properties of TCP and its children, 
+          type: array
+          items:
+            $ref: './tcp/tcp.yaml#/components/schemas/App.Tcp'  
+          x-field-uid: 1
+        http:
+          description: >-
+            The properties of HTTP and its children, 
+          type: array
+          items:
+            $ref: './http/http.yaml#/components/schemas/App.Http'
+          x-field-uid: 2
+        ssl:
+          description: >-
+            SSL configuration
+          type: array
+          items:
+            $ref: './ssl/ssl.yaml#/components/schemas/App.Ssl'
+          x-field-uid: 3
+        traffic:
+          description: >-
+            Traffic configuration of endpoints 
```

Comment:
Need to be more precise as to how this differs from 'flows'.  That is also traffic configuration.